_damo_deprecated.py

```diff
@@ -1,9 +1,9 @@
 # SPDX-License-Identifier: GPL-2.0
 
-'''
+"""
```

Comment:
I personally prefer single quote.  Could we change the configuration for that?

Reply:
I also read that, but I cannot still strongly agree to their argument.  I may need more time to deal with my mean preference.